src/components/AddWithdraw/components/AddWithdrawCountriesList.tsx

```
@@ -10,29 +10,174 @@
+        if (!isUserKycVerified) {
+            const detailsToSave = { ...rawData }
+
+            // we're caching the bank details in local storage so we don't lose them
+            // during the kyc flow. we don't need to cache the user's name and email,
+            // so we remove them from the object we're about to save.
+            //
+            // we use @ts-ignore here because typescript would normally complain
+            // that we're deleting properties that are defined on the formdata type.
+            // in this case, it's safe because we're just preparing the data for caching.
+            // @ts-ignore
+            delete detailsToSave.fullName
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)